src/libsyntax/parse/lexer/mod.rs

```diff
@@ -175,19 +176,6 @@ impl<'a> StringReader<'a> {
         self.ch.is_none()
     }
 
-    fn fail_unterminated_raw_string(&self, pos: BytePos, hash_count: u16) {
-        let mut err = self.struct_span_fatal(pos, pos, "unterminated raw string");
-        err.span_label(self.mk_sp(pos, pos), "unterminated raw string");
-
-        if hash_count > 0 {
-            err.note(&format!("this raw string should be terminated with `\"{}`",
```

Comment:
seems like this error hint is lost?

Reply:
Yeah, the raw string handling will be reworked, see #60679

Reply:
might also be worth trying to capture *too many* closing hashes with a better note? instead of falling back on
```
error: expected one of `.`, `;`, `?`, or an operator, found `#`
```